WalletWasabi.Packager/Content/Osx/App/Contents/Info.plist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>LSArchitecturePriority</key>
+	<array>
+		<string>x86_64</string>
+	</array>
+	<key>CFBundleIconFile</key>
+	<string>WasabiLogo.icns</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>?</string>
+	<key>CFBundleVersion</key>
+	<string>?</string>
+	<key>CFBundleExecutable</key>
+	<string>wassabee</string>
+	<key>CFBundleName</key>
+	<string>Wasabi Wallet</string>
+	<key>CFBundleIdentifier</key>
+	<string>zksnacks.wasabiwallet</string>
+	<key>NSHighResolutionCapable</key>
+	<true/>
+	<key>NSAppleScriptEnabled</key>
```

Comment:
What is this for?

Reply:
Apple script is that you can run this from AppleScript. You can do various things with that for example timing start of the application or anything. Who knows who is using this. Better to let it be there.